TicketSwap/solar
# feat: add Bookmark icon

# Description

This PR will add a bookmark icon to Solar.

## How to test

- Checkout this branch
- `yarn storybook`
- Verify the Bookmark icon is showing up in the iconography tab

## Screenshots

![Screenshot 2022-07-10 at 20 54 06](https://user-images.githubusercontent.com/14276144/178158234-da0b3d8f-92bc-4ecb-8b8f-e2c360902968.png)
